docs/workflow/ci/failure-analysis.md

```diff
@@ -0,0 +1,108 @@
+    Build Information
+    Build: https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_build/results?buildId=242380
+    Build error leg or test failing: Build / linux-arm64 Release AllSubsets_Mono_Minijit_RuntimeTests minijit / Build Tests
+    Pull request: https://github.com/dotnet/runtime/pull/84716
+    <!-- Error message template  -->
+    ## Error Message
+    Fill the error message using [known issues guidance](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssues.md#how-to-fill-out-a-known-issue-error-section).
+
+    ```json
+    {
+        "ErrorMessage": "",
+        "BuildRetry": false,
+        "ErrorPattern": "",
+        "ExcludeConsoleLog": false
+    }
+    ```
+    ````
+    It already contains most information necessary. *The most important piece of information for you to fill out is the json blob*.
+
+    - You can add the string you identified as uniquely identifying the issue into the `ErrorMessage` field. In case you need to use a regex, use the `ErrorPattern` field instead (this is a limitted, single-line, non-backtracking regex described [here](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssues.md#regex-matching)). This regex needs to be appropriately escaped - check the [arcade known issues](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssues.md#filling-out-known-issues-json-blob) documentation for a good guide on proper regex and JSON escaping.
+    - The field `ExcludeConsoleLog` should be `true` as much as possible. Setting it to false will mean all failures appear in the same log and since you didn't exclude the overall logs, all failures in the collection will get attributed to this one's issue bucket and errors will not be investigated properly. Due to limitations in Known Issues around rate limiting and xUnit resiliency, setting `ExcludeConsoleLog=false` is necessary in two scenarios:
+      + Nested tests as reported to Azure DevOps. Essentially this means theory failures, which look like this when reported in Azure DevOps: ![xUnit theory seen in azure devops](theory-azdo.png).
+        Adding support for this requires too many API calls, so using the console log here is necessary.
+      + Native crashes in libraries also require using the console log. This is needed as the crash corrupts the test results to be reported to Azure DevOps, so only the console logs are left.
+    - Optionally you can add specifics as needed like leg, configuration parameters, available dump links.
+
+Once the issue is open, feel free to rerun the `Build Analysis` check and the issue should be recognized as know if all was filed correctly. there are some known limitations as previously described. It also has no support at looking at fields outside the error message, the stacktrace, and the console log in the helix queue, so this approach doesn't yet cover devices too well yet.
```

Comment:
I would rewrite "...feel free to rerun the `Build Analysis` check and the issue should be recognized as know if all was filed correctly. there are some known limitations as previously described" as:
"...feel free to rerun the `Build Analysis` check and the issue you just filed should be recognized as known, if everything was done correctly. However, it is important to remember there are some known limitations in this system, as previously described"